src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -864,34 +803,45 @@ public static function insertCompletePost(array $item, array $meta = [], $tags =
         return $item['revision_id'];
     }
 
-    /**
-     * Inserts a new category into the database
-     *
-     * @param array $item The data for the category to insert.
-     * @param array $meta The metadata for the category to insert.
-     *
-     * @return int
-     */
-    public static function insertCategory(array $item, array $meta = null): int
+    public static function insertCategory(array $item, ?array $meta = null): int
     {
-        // get database
-        $database = BackendModel::getContainer()->get('database');
+        $entityManager = BackendModel::get(
+            'doctrine.orm.default_entity_manager'
+        );
 
-        // meta given?
-        if ($meta !== null) {
-            $item['meta_id'] = $database->insert('meta', $meta);
+        if ($meta === null) {
+            $meta = BackendModel::get('fork.repository.meta')
+                                ->find($item['meta_id']);
+        } else {
+            $meta = new Meta(
+                $meta['keywords'],
+                (isset($meta['keywords_overwrite'])) ? $meta['keywords_overwrite'] : false,
```

Comment:
for this one and all the ones below
`$meta['keywords_overwrite'] ?? false`

Reply:
Fixed in e1a30444f2858439c12fb88659292f551b1e924a